src/test/kotlin/com/manimdsl/stylesheet/StylesheetUnitTests.kt

```diff
@@ -53,8 +54,8 @@ class StylesheetUnitTests {
         assertThat(stack1Style.creationStyle, `is`("GrowFromCenter"))
 
         val stack1AnimationStyle = stack1Stylesheet.getAnimatedStyle("stack1", StackValue(EmptyMObject, Stack()))!!
-        assertThat(stack1AnimationStyle.borderColor, `is`("BLUE"))
-        assertThat(stack1AnimationStyle.textColor, `is`("RED"))
+        assertThat(stack1AnimationStyle.borderColor, `is`("RED"))
```

Comment:
Can you change this to `RED` and `RED`? I just checked the [stylesheet](https://github.com/ManimDSL/ManimDSLCompiler/blob/7f4f6ecb0183c629699401d88e6a55aac38b1611/src/test/testFiles/stylesheet/stackTypeStylesheet.json) on this branch and that's what the colours should be